README.md

```diff
@@ -175,32 +175,39 @@ More performance fine-tuning is possible when using Kafka, ie. with `spec.deploy
-- The metrics server running in the processor (`flowlogs-pipeline`) can listen using TLS, via `spec.processor.metrics.server.tls`.
-- The Console plugin server always uses TLS.
+To make autorized queries to Loki using the Loki Operator, NetObserv must be configured as such:
 
-To make authenticated queries to Loki, when using the Loki Operator, there are two options:
+- set `spec.loki.authToken` to `FORWARD` in the `FlowCollector` resource. The console plugin will forward the logged-in Console user token to Loki.
+- install `ClusterRole` and `ClusterRoleBinding` for NetObserv: [role.yaml](https://github.com/netobserv/documents/blob/main/examples/loki-stack/role.yaml).
+- To get access to the flow logs, users must have a `ClusterRoleBinding` for the `netobserv-reader` role. E.g: [rolebinding-user-test.yaml](https://github.com/netobserv/documents/blob/main/examples/loki-stack/rolebinding-user-test.yaml).
 
-- per-user auth: set `spec.loki.authToken` to `FORWARD` in the `FlowCollector` resource. The console plugin will forward the logged-in Console user token to Loki. To get access to the flow logs, users must have the following permission:
+Or, using the CLI for user `test`:
 
-```yaml
-- apiGroups:
-  - 'loki.grafana.com'
-  resources:
-  - network
-  resourceNames:
-  - logs
-  verbs:
-  - 'get'
+```bash
+oc adm policy add-cluster-role-to-user netobserv-reader test
 ```
 
-- per-app auth, ie. with NetObserv components having access to the logs using their own service account, regardless of the user permissions. For this mode, set `spec.loki.authToken` to `HOST` and setup `ClusterRole` for `flowlogs-pipeline` and `netobserv-plugin`, e.g: [role.yaml](https://github.com/netobserv/documents/blob/main/examples/loki-stack/role.yaml).
+More information about multi-tenancy can be found on [this page](https://github.com/netobserv/documents/blob/main/loki_operator.md#netobserv-configuration).
+
+Note that multi-tenancy is not possible when not using the Loki Operator.
+
+#### Network Policy
 
+For a production deployment, it is also highly recommended to lock down the `netobserv` namespace (or wherever NetObserv is installed) using network policies.
+An example of such a network policy is [provided here](https://github.com/netobserv/documents/blob/main/examples/lockdown-netobserv.yaml).
+
+#### Communications
+
+By default, communications between internal components are not secured. Note that, when using the Loki Operator, securing communication with TLS is necessary. There are several places where TLS can be set up:
+
+- Connections to Loki (from the processor `flowlogs-pipeline` and from the Console plugin), by setting `spec.loki.tls`.
+- With Kafka (both on producer and consumer sides), by setting `spec.kafka.tls`. Mutual TLS is supported here.
+- The metrics server running in the processor (`flowlogs-pipeline`) can listen using TLS, via `spec.processor.metrics.server.tls`.
```

Comment:
```suggestion
- The metrics server (running in the processor `flowlogs-pipeline`) can listen using TLS, via `spec.processor.metrics.server.tls`.
```

Reply:
thanks @jpinsonneau 
I didn't take that one: there's several metrics servers (in flp but also in console plugin and in the operator) so I think parenthesis should be kept just around "flowlogs-pipeline"